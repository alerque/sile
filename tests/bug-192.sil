\begin[papersize=a6]{document}
\nofolios
\font[family=Noto Naskh Arabic,language=urd,size=18pt]
\hbox\skip[height=95%fh]
کب سے ہیں اِسی کوشش میں اب\break\skip[height=2ex]
کب سے ہیں اِسی کوشش میں اب\break
\end{document}
