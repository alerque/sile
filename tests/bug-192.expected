Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	20.9764
Mx 	34.8819
My 	381.3449
Set font 	Noto Naskh Arabic;18;400;;normal;;RTL
T	40 39	(اب)
Mx 	57.8320
T	365 408 494	(میں)
Mx 	87.8309
T	460 461 502 354	(کوشش)
Mx 	138.6335
T	406 458 80 39	(اِسی)
Mx 	171.8931
T	365 408 369	(ہیں)
Mx 	198.6489
T	374 458	(سے)
Mx 	224.7982
T	434 354	(کب)
New page
Mx 	34.8819
My 	33.9315
T	40 39	(اب)
Mx 	57.8320
T	365 408 494	(میں)
Mx 	87.8309
T	460 461 502 354	(کوشش)
Mx 	138.6335
T	406 458 80 39	(اِسی)
Mx 	171.8931
T	365 408 369	(ہیں)
Mx 	198.6489
T	374 458	(سے)
Mx 	224.7982
T	434 354	(کب)
End page
Finish
